<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
      <item>
       <widget class="QWidget" name="widget_list" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5,5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QComboBox" name="v_d_comboBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>视频列表</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>设备列表</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="new_file">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>resources/op_icon/file.png</normaloff>resources/op_icon/file.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="add_workspace">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>25</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>resources/op_icon/add.png</normaloff>resources/op_icon/add.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTreeWidget" name="video_tree">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="work_list">
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="ori_video">
         <attribute name="title">
          <string>原始/实时视频</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,2">
            <item>
             <widget class="QWidget" name="video_widget" native="true">
              <layout class="QGridLayout" name="gridLayout_4">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_2" stretch="9,9,1,1">
                 <property name="spacing">
                  <number>5</number>
                 </property>
                 <item>
                  <widget class="QVideoWidget" name="player" native="true">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="autoFillBackground">
                    <bool>false</bool>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">
        background: rgb(0, 0, 0);</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="camera">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">
        background: rgb(0, 0, 0);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="10,1">
                     <item>
                      <widget class="QSlider" name="video_slider">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="video_time">
                       <property name="text">
                        <string>0:0</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="play_pause">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset>
                     <normaloff>resources/play_ico.png</normaloff>resources/play_ico.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="video_label_widget" native="true">
              <layout class="QGridLayout" name="gridLayout_6">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_3" stretch="10,0,5">
                 <item>
                  <widget class="QTextBrowser" name="video_info"/>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1">
                   <item>
                    <widget class="QLabel" name="video_label">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>备注</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="save_label">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="mouseTracking">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset>
                       <normaloff>resources/op_icon/save.png</normaloff>resources/op_icon/save.png</iconset>
                     </property>
                     <property name="autoRepeat">
                      <bool>false</bool>
                     </property>
                     <property name="autoDefault">
                      <bool>false</bool>
                     </property>
                     <property name="default">
                      <bool>false</bool>
                     </property>
                     <property name="flat">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QTextEdit" name="video_label_edit">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="act_identify">
         <attribute name="title">
          <string>行为识别</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="5,2">
            <item>
             <widget class="QWidget" name="video_widget_2" native="true">
              <layout class="QGridLayout" name="gridLayout_7">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_4" stretch="9,0,1,0">
                 <item>
                  <widget class="QLabel" name="camera_2">
                   <property name="styleSheet">
                    <string notr="true">
        background: rgb(0, 0, 0);</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QVideoWidget" name="player_2" native="true">
                   <property name="styleSheet">
                    <string notr="true">
        background: rgb(0, 0, 0);</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="10,1">
                     <item>
                      <widget class="QSlider" name="video_slider_2">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="video_time_2">
                       <property name="text">
                        <string>0:0</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <widget class="QPushButton" name="start_identify">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string>启动识别</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="play_pause_2">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset>
                       <normaloff>resources/play_ico.png</normaloff>resources/play_ico.png</iconset>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="video_label_widget_2" native="true">
              <layout class="QGridLayout" name="gridLayout_8">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_5" stretch="5,0,5">
                 <item>
                  <widget class="QListWidget" name="action_list"/>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1">
                   <item>
                    <widget class="QLineEdit" name="item_search"/>
                   </item>
                   <item>
                    <widget class="QPushButton" name="item_search_button">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="mouseTracking">
                      <bool>false</bool>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset>
                       <normaloff>resources/op_icon/search.png</normaloff>resources/op_icon/search.png</iconset>
                     </property>
                     <property name="autoRepeat">
                      <bool>false</bool>
                     </property>
                     <property name="autoDefault">
                      <bool>false</bool>
                     </property>
                     <property name="default">
                      <bool>false</bool>
                     </property>
                     <property name="flat">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QListWidget" name="search_result"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="abn_identify">
         <attribute name="title">
          <string>异常行为检测</string>
         </attribute>
         <widget class="QSplitter" name="splitter">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>20</y>
            <width>256</width>
            <height>384</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>UpArrowCursor</cursorShape>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTextEdit" name="textEdit_2"/>
          <widget class="QTextEdit" name="textEdit"/>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="system_set">
    <property name="title">
     <string>系统设置</string>
    </property>
    <addaction name="video_select"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="device_manager">
    <property name="title">
     <string>设备管理</string>
    </property>
   </widget>
   <widget class="QMenu" name="model_manager">
    <property name="title">
     <string>模型设置</string>
    </property>
   </widget>
   <addaction name="system_set"/>
   <addaction name="device_manager"/>
   <addaction name="model_manager"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="video_select">
   <property name="icon">
    <iconset>
     <normaloff>resources/file.ico</normaloff>resources/file.ico</iconset>
   </property>
   <property name="text">
    <string>视频目录选择</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
